init-pants/action.yaml

```diff
@@ -105,8 +105,16 @@ runs:
     - name: Tell pants to use CI config
       if: ${{ inputs.pants-ci-config != '' }}
       shell: bash
+      env:
+        PANTS_CONFIG_FILES: ${{ inputs.pants-ci-config }}
       run: |
-        echo "PANTS_CONFIG_FILES=${{ inputs.pants-ci-config }}" >> ${GITHUB_ENV}
+        if [ -f ${PANTS_CONFIG_FILES} ] || [ "${PANTS_CONFIG_FILES}" != "pants.ci.toml" ]; then
```

Comment:
Is there any way of telling if the input was set explicitly? Ideally we would distinguish between "user didn't specify any file" and "user specified pants.ci.toml" explicitly?

Reply:
I suppose we could change the default to something like `DEFAULT` and then if `== "DEFAULT"` and `pants.ci.toml` exists, use that. Would that be better?

Reply:
Oh yeah, that would work!

Reply:
Changed. How does it look now?